### PR TITLE
Photo gallery with DevConfCZ2020 pictures

### PR DESCRIPTION
**Pull request by iranzo**
**What this PR does / why we need it**:

Show DevConf.cz KubeVirt presence

**Does this PR fix any issue?** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
> Fixes #

**Special notes for your reviewer**:

Article preview at: https://deploy-preview-450--kubevirtio.netlify.com/galleries/2020-01-31-devconfcz2020-in-pictures

### CODE REVIEW

File: _posts/2020-01-31-DevConfCZ2020-in-pictures.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+category: gallery
+comments: true
+title: DevConf.CZ 2020 in pictures
+pub-date: January 31
+pub-year: 2020
+galleria: devconfcz2020.json
+
+---
+
+Here are some of the pictures of KubeVirt presence at [DevConfCZ 2020](https://www.devconf.info/cz/).
```
Comment (codificat) [inline _posts/2020-01-31-DevConfCZ2020-in-pictures.md]: If we publish a blog post we probably want to say a bit more about KubeVirt's presence at DevConf.

I'll take a stab at it when I find the time, and/or ping other attendees to provide their input

**Comment by codificat** [comment]
/hold
until we have some content

File: _posts/2020-01-31-DevConfCZ2020-in-pictures.md (hunk 1 above)
Comment (iranzo) [reply _posts/2020-01-31-DevConfCZ2020-in-pictures.md]: That's why we left this on hold until someone could either write a blog post and incorporate the gallery or either write post on one side and gallery on another and probably link between both

**Comment by ptrnull** [comment]
@codificat we don't need content to publish the photo gallery, do we?

**Comment by fabiand** [comment]
@codificat can you or somebody review and merge this pr?

**Comment by mazzystr** [comment]
This is not going to be easily mergeable since we don't hear from Iranzo much anymore.  Closing.

**Comment by iranzo** [comment]
> This is not going to be easily mergeable since we don't hear from Iranzo much anymore. Closing.

This was ready in January, you can easily get this rebased and updated so that in can be mergeable again.

Regards

**Comment by mazzystr** [comment]
@iranzo Not sure what happened here.  Your fork seems to have disappeared.

**Pull request closed without merging**